### ISSUE TITLE
Change resubmit to use JAWS

### ISSUE THREAD

**Issue by kaijli** [microbiomedata/nmdc_automation#600]
Change the resubmit logic to utilize JAWS resubmit instead of making a new JAWS ID for each automation resubmit. This way, we don't have to deal with 3 consecutive JAWS IDS for one job that keeps failing.
